rke/templates/coredns.go

```diff
@@ -97,11 +97,14 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
     kubernetes.io/name: "CoreDNS"
 spec:
-  replicas: 1
```

Comment:
Why are we removing replicas?

Reply:
yeah the autoscaler determines replicas and this field doesn't get used